sklearn/cross_decomposition/pls_.py

```diff
@@ -311,7 +314,8 @@ def fit(self, X, Y):
             y_scores = np.dot(Yk, y_weights) / y_ss
             # test for null variance
             if np.dot(x_scores.T, x_scores) < np.finfo(np.double).eps:
-                warnings.warn('X scores are null at iteration %s' % k)
+                warnings.warn('X scores are null at iteration %s' % k,
```

Comment:
It sounds like the warning is in here to note underflow, and I think we should not call this a convergence warning.  In particular, we give no way for the user to improve upon this situation.

Reply:
Ok. I changed it back. I guess the same then applies to the warning about the Y residual here:
https://github.com/jotasi/scikit-learn/blob/14509c2431ae9c683fdd1d36b34d8de4dfbd7b24/sklearn/cross_decomposition/pls_.py#L288-L290
I changed that back as well.